ci-operator/config/openshift/cluster-api-provider-alibaba/openshift-cluster-api-provider-alibaba-main.yaml

```diff
@@ -21,6 +21,20 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
   to: alibaba-machine-controllers
+promotion:
+  disabled: true
```

Comment:
Why would you want the promotion to be disabled?

Reply:
It is the same for all other cluster api providers because promotion is disabled by default when you use `make new-repo`... as I understand, it just disables automated promotion, but we need to consult with the CI team.

[AWS](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master.yaml#L23)
[Azure](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml#L26)
[GCP](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-gcp/openshift-cluster-api-provider-gcp-master.yaml#L22)